_component_design/time-selector.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Time Selector
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: The time selector enables the user to select a time from a list of values. The time selector is often accompanied by a date selector that a user can use to select a date.
+preview-image: preview-images/time-selector.svg
+resource: true
+status: stable
+last-modified: 2018-11-07
+helix-ui-css: false
+helix-ui-javascript: false
+---
+
+{% include toc.html %}
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use the time selector when you want to enable a user to select a time from a list of values. The time selected includes an hour value and a minute value. The hour value can include 12-hour clock values, or 24-hour clock values. Minutes are grouped into 10-minute, 15-minute, or 30-minute intervals. For example, a user can select a time of **1:00 PM**, **1:10 PM**, **1:15 PM**, or **1:30 PM**, but cannot select any other minute values.
+
+If you want the user to select a time from a limited list of predetermined times, consider using [radios]({{site.baseurl}}\components\radios.html).
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+Consider the following best practices when you construct a time selector:
+
+- Depending on the use case, provide 10-minute, 15-minute, or 30-minute increments in a time selector.
+- If you are using a 12-hour clock, add **AM** and **PM** in the drop-down selector. **AM** and **PM** are not required for a 24-hour clock.
+- Construct the time selector so that the user can tab to it, which focuses the time selector without triggering the drop-down selector.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Specifications
+
+Consider the following specifications when you construct a time selector.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Composition
+
+A time selector is comprised of the following elements:
+
+- **Drop-down selector (required)**: All possible minute and hour combinations appear on a drop-down list that the user accesses by clicking the drop-down selector. The drop-down selector can be activated by keyboard input or when the user clicks anywhere on the time selector. See [drop-down selector]({{site.baseurl}}\components\drop-down-selector.html) for construction guidelines.
```

Comment:
"all possible" opens the door for a massively crazy long list. maybe remove the all encompassing nature of the sentence and just state the the dropdown selector contains selectable times.

Reply:
Gotcha! Done.